### ISSUE TITLE
Hi there 

### ISSUE THREAD

**Issue by ucalyptus** [DanielhCarranza/Machine-Learning-in-Health#1]
Can you tell me the issue? is it the make directory issue?

**Comment by DanielhCarranza** [comment]
No,  I was trying to apply the trasformations one by one in pure pytorch
based on the fastai scans. I going to share with you the new version of my
colab.
https://colab.research.google.com/drive/1eBiWGZFG4-YK9LkqTulu6wAd18WQYjVt

My plan is to use XResnet3D as a backbone to the Unet3D (fastai version
modified).
